main/src/library/pkger/Manifest.flix

```diff
@@ -0,0 +1,287 @@
+namespace Manifest {
+    pub enum License {
+        case MIT
+        case GPLv3
+        case Apache2
+        case BSD
+        case Other(String)
+    }
+
+    pub type alias Author = { name: String
+                            // Is there an email datatype, or is it just commonly a String?
+                            , email: String
+                            }
+
+    pub type alias Package = { name: String
+                             , version: SemVer.SemVer
+                             , authors: List[Author]
+                             , description: String
+                             // This could be mandatory
+                             , license: Option[License]
```

Comment:
I think the license should be mandatory. We cannot really have code without a license.